child_update_picture_date/wizard/__init__.py

```diff
@@ -0,0 +1,12 @@
+# -*- encoding: utf-8 -*-

```

Comment:
We put it in a separate module because we needed to perform the update only once, and then we can uninstall the module and never use this code anymore.